src/shared/views/ChangeDetection/TaskList.js

```diff
@@ -40,6 +40,7 @@ type Props = {
     closeTilePopup: () => void,
     openTilePopup: () => void,
     zoomLevel: number,
+    continueMapping: boolean,
```

Comment:
Could you maybe rename this to `canContinueMapping`? I think it makes it clearer what the variable means, no?

Reply:
good idea. will rename this in all places where it occurs